### ISSUE TITLE
invalid istio-sidecar.deb in istio-release-0.8-20180515-17-26

### ISSUE THREAD

**Issue by kidiyoor** [istio/istio#5654]
```
Welcome to Ubuntu 16.04.4 LTS (GNU/Linux 4.13.0-1012-gcp x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  Get cloud support with Ubuntu Advantage Cloud Guest:
    http://www.ubuntu.com/business/services/cloud
12 packages can be updated.
1 update is a security update.
*** System restart required ***

gauthamvk@prelim:~$ ls
cluster.env  istio-sidecar.deb  istio.VERSION  kubedns  root-cert.pem  setupIstioVM.sh

gauthamvk@prelim:~$ sudo dpkg -i istio-sidecar.deb
dpkg-deb: error: 'istio-sidecar.deb' is not a debian format archive
dpkg: error processing archive istio-sidecar.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 istio-sidecar.deb
```

```
Linux prelim-2 4.9.0-6-amd64 #1 SMP Debian 4.9.82-1+deb9u3 (2018-03-02) x86_64
The programs included with the Debian GNU/Linux system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.
Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.

gauthamvk@prelim-2:~$ sudo dpkg -i istio-sidecar.deb
dpkg: error processing archive istio-sidecar.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 2 package 'istio-sidecar':
 error in 'Version' field string 'release-0.8-20180515-17-26': version number does not start with digit
Errors were encountered while processing:
 istio-sidecar.deb
gauthamvk@test1:
```

**Comment by hklai** [comment]
I am hoping we can decouple the monthly version scheme from these dailies.

Perhaps we can just use the datetime for master...

i.e. 0.8-20180515-17-26 for "release-0.8" and just 20180515-17-26 for master.

**Comment by ldemailly** [comment]
if that doesn't make deb complain that sounds fine with me

**Comment by rkpagadala** [comment]
Fixed with #5698
